central/detection/service/service_impl.go

```diff
@@ -220,10 +225,19 @@ func (s *serviceImpl) enrichAndDetect(ctx context.Context, enrichmentContext enr
 		EnforcementOnly: enrichmentContext.EnforcementOnly,
 	}
 
+	var appliedNetpols *augmentedobjs.NetworkPoliciesApplied
+	appliedNetpols, err = s.getAppliedNetpolsForDeployment(ctx, enrichmentContext, deployment)
+	if err != nil {
+		log.Warnf("Could not find applied network policies for deployment %s. Continuing with deployment enrichment.", deployment.GetName())
```

Comment:
Sorry, looking at this after some break makes me see it from a different perspective :)

Would it make sense to include the `err` message in that log line? I don't know what kind of errors are returned from that func, but if there is more than one type of error possible, then I think it maybe useful to print it.

Reply:
Done!